src/StateChannelEventListener.ts

```diff
@@ -105,15 +107,17 @@ class StateChannelEventListener {
             handler: (logObj: any) => {
                 const {
                     channelId,
-                    dispute,
+                    disputeConfirmation,
                     disputeCreationTimestamp,
                     isFinal,
                     windowCreationTimestamp
                 } = logObj.args;
 
                 this.eventHandler.onDisputeCommitted(
                     channelId,
-                    Codec.convertEthersResultToObject(dispute),
+                    (disputeConfirmation as Result).toObject(
+                        true
+                    ) as DisputeConfirmationStruct,
```

Comment:
is this intentional or forgot to change back to our custom implementation?

Reply:
I changed to try, it worked. I didn't change back.

Edit:
Changed back